packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -311,9 +325,16 @@ function and(
 			desiredDocSchema.runtime[key],
 		);
 	}
+
+	const minVersionForCollab = getGreaterMinVersionForCollab(
```

Comment:
This should get the Lower one.  "and" is about what satisfies both requirements, and applies to this session at the start.  Then as soon as changes are made to the document, `maybeSendSchemaMessage` is called, and it gives an op that updates the document schema to the "or" result, which in this case should be the Greater one.  This is how it will advance over time.

Reply:
Actually I think it should just return current.

Reply:
And this would apply to all `info` type schema properties.  The info can't be combined, it just stays as-is for this session, until the op goes out to bump it to the new value

Reply:
I do think we should rename "and" and "or" at some point.  This logic isn't commutative, which would be surprising.